### ISSUE TITLE
A way to update only the outdated frameworks

### ISSUE THREAD

**Issue by sprint84** [Carthage/Carthage#1297]
* `carthage version`: 0.16.2
* `xcodebuild -version`: Xcode 7.3.1
Build version 7D1014
* Are you using `--no-build`? No
* Are you using `--no-use-binaries`? No
* Are you using `--use-submodules`? No

Hi,

is there a command that I can use that will skip the build step for Frameworks that are already up-to-date?
I noticed that carthage added the `outdated` command, which is great to find the outdated dependencies.
And I can run `carthage update framework_name` to achieve that.

Basically what I am asking is if there is a way to combine both, such as `carthage update outdated`.
This would speed up a lot automated processes such as fastlane tools.

Thanks

**Comment by mdiep** [comment]
There is not currently. But only building updated dependencies might be a good idea.

**Comment by grangej** [comment]
I'm going to fork and work on this.... if my team and I do not find a solution for reducing build times there might be mutiny ;)

**Comment by ikesyo** [comment]
Is this a duplicate of #975?

**Comment by mdiep** [comment]
Yes it is!

**Comment by sprint84** [comment]
If it's a duplicate, is it implemented?
`carthage help update` does not show that option.
